rest-api-spec/build.gradle

```diff
@@ -82,6 +82,8 @@ tasks.named("yamlRestTestV7CompatTransform").configure { task ->
   task.skipTest("search/370_profile/fetch source", "profile output has changed")
   task.skipTest("search/370_profile/fetch nested source", "profile output has changed")
   task.skipTest("search/240_date_nanos/doc value fields are working as expected across date and date_nanos fields", "Fetching docvalues field multiple times is no longer allowed")
+  task.skipTest("search/110_field_collapsing/field collapsing and scroll", "Error message has changed")
```

Comment:
@jakelandis Do you think it's worth having a transform to deal with changes like this so we don't have to mute compatibility tests for this reason?

Reply:
oh right, I did not realize that was an option, I will do that.

Reply:
@javanna FYI, there is no such transform for `catch` clauses. We'd have to add one. You're welcome to do this. I was just wondering if we think this is a common enough scenario to justify doing so.

Reply:
thanks for pointing that out. I was already doubting myself if changing the error message was worth it. It was changed marginally hence I reverted that bit and left the message as it previously was to avoid bw comp issues.